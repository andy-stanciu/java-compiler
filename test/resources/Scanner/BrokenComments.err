Unexpected character '/' on line 1 at column 1 of input.
Unexpected character '/' on line 9 at column 1 of input.
Unexpected character '/' on line 11 at column 7 of input.
Unexpected character '/' on line 14 at column 14 of input.
Unexpected character '/' on line 19 at column 1 of input.
Unexpected character '/' on line 19 at column 8 of input.
Unexpected character '/' on line 19 at column 10 of input.
Unexpected character '/' on line 19 at column 14 of input.
Unexpected character '/' on line 19 at column 20 of input.
Unexpected character '/' on line 19 at column 27 of input.
Unexpected character '/' on line 19 at column 32 of input.
