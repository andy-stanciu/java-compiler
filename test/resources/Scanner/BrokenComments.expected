<UNEXPECTED(/)> TIMES THIS ID(is) ID(broken) ID(very) ID(broken) <UNEXPECTED(/)> <UNEXPECTED(/)> TIMES ID(should) ID(have) <UNEXPECTED(/)> TIMES TIMES TIMES TIMES TIMES TIMES TIMES TIMES ID(many) ID(problems) <UNEXPECTED(/)> <UNEXPECTED(/)> <UNEXPECTED(/)> <UNEXPECTED(/)> <UNEXPECTED(/)> <UNEXPECTED(/)> <UNEXPECTED(/)> ID(done) DOT 